Chefs/Presentation/RecipeDetailsModel.cs

```diff
@@ -32,39 +34,51 @@ public RecipeDetailsModel(
 		_shareService = shareService;
 
 		Recipe = recipe;
+		_recipeFeed = new(recipe, _recipeService, _userService);
 	}
 
 	public Recipe Recipe { get; }
-	public IState<bool> IsFavorited => State.Value(this, () => Recipe.IsFavorite);
 
-	public IState<User> User => State.Async(this, async ct => await _userService.GetById(Recipe.UserId, ct))
-		.Observe(_messenger, u => u.Id);
+	public IFeed<RecipeInfo> RecipeDetails => _recipeFeed.Feed;
 
-	public IFeed<User> CurrentUser => Feed.Async(_userService.GetCurrent);
-	public IListFeed<Ingredient> Ingredients => ListFeed.Async(async ct => await _recipeService.GetIngredients(Recipe.Id, ct));
-	public IListFeed<Step> Steps => ListFeed.Async(async ct => await _recipeService.GetSteps(Recipe.Id, ct));
+	public async ValueTask Like(Review review, CancellationToken ct)
+		=> await _recipeService.LikeReview(review, ct);
 
-	public IListState<Review> Reviews => ListState
-		.Async(this, async ct => await _recipeService.GetReviews(Recipe.Id, ct))
-		.Observe(_messenger, r => r.Id);
+	public async ValueTask Dislike(Review review, CancellationToken ct)
+		=> await _recipeService.DislikeReview(review, ct);
 
-	public async ValueTask Like(Review review, CancellationToken ct) =>
-		await _recipeService.LikeReview(review, ct);
+	public async ValueTask LiveCooking(RecipeInfo recipeDetails)
+		=> await _navigator.NavigateDataAsync(this, data: new LiveCookingParameter(recipeDetails.Recipe, recipeDetails.Steps));
 
-	public async ValueTask Dislike(Review review, CancellationToken ct) =>
-		await _recipeService.DislikeReview(review, ct);
+	public async ValueTask Favorite(Recipe recipe, CancellationToken ct)
+		=> await _recipeService.Favorite(recipe, ct);
```

Comment:
I'm not able to hit this line `=> await _recipeService.Favorite(recipe, ct);` when placing a breakpoint.

Reply:
good catch, fixed